### PR TITLE
Python code for CSES Subtree Queries in Euler Tour module

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/cses-1706.mdx
Comment (bqi343) [inline solutions/cses-1706.mdx:1]: delete?

Comment (brebenelmihnea) [reply solutions/cses-1706.mdx:1]: ?

Comment (bqi343) [reply solutions/cses-1706.mdx:1]: why is this file part of the PR?

Comment (brebenelmihnea) [reply solutions/cses-1706.mdx:1]: cses-1706.mdx ?

Comment (bqi343) [reply solutions/cses-1706.mdx:1]: yes

Comment (bqi343) [reply solutions/cses-1706.mdx:1]: yes. what does cses-1706 have to do with Tree_Euler.mdx though?

**Comment by brebenelmihnea** [comment]
Oh, my mistake I deleted the wrong file. I will delete this PR and make another one.

**Pull request closed without merging**